.travis.yml

```
@@ -193,9 +193,9 @@ jobs:
 # ------------------------
 stages:
   - name: Unit Tests & Linters
-    if: type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
+    if: type =~ /(cron|pull_request)/ OR (type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/)
   - name: Integration tests
-    if: deactivated = false AND type =~ /(cron|push|pull_request)/ AND branch =~ /^master|[0-9]+-(dev|stable)$/
+    if: deactivated = false AND type =~ /(cron|pull_request)/ OR (type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/)
```

If I understand you want a environment variable to enable/disable integration tests. So the right way to do that is:
```suggestion
    if: env(DEACTIVATED) = false AND type =~ /(cron|pull_request)/ OR (type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/)
```

---

I don't care if it can be controlled with an environment variable, I just want a test that is always false to temporarily deactivate functional tests.

The goals here are:
* to make PRs easily reviewable: red PR should not be merged, even with Kuzzle 2.0 in a "in progress" state
* to have an easy way to reactivate functional tests once Kuzzle 2.0 is ready to be functionally tested: you can do that by simply undoing this change

---

I understood the main goal of this variable. But this not a valid Travis syntax and it's better to have it as an environment variable to disable/enable tests from Travis Web UI.